firmware/application/freqman.cpp

```diff
@@ -48,7 +48,7 @@ options_t freqman_entry_bandwidths[ 4 ] = {
     { //WFM
         { "200k" , 0 },
         { "180k" , 1 },
-        { "40k"  , 2 },
+        { " 40k" , 2 },
```

Comment:
Didn't see that one when updating. Thanks for the fix, I'll update the doc about the format for it
EDIT:
After searching why only that entry needed a space, it's in fact because the options_t field for WFM in analog_audio_app.cpp is defined with only 3 character wide instead of 4. 
We can commit like this for now, but I would prefer it to be "40k" and the option_t having the good size, so in freqman files when referencing it it can be written 'bw=40k,' or 'bw=40k ,' instead of a forced 'bw= 40k' (notice the obligatory space before).
That, or I can try to add trimming to read values in the freqman loader.

Reply:
As discussed with @Brumi-2021, it's a two lines modification adjustment, we can start by providing the functionalities like it is, and fix the pesky space thing on a minor PR later